### DISCUSSION TITLE
Prisma, filter by the length of an array

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#14476] [Q&A]
Hi guys !
My prisma schema is built with two main models : User and Book.
i have something like : 
model User { 
    books    Book[]
}
and i want to get the number of users who have more than 0 books. It seems simple but it isn't. I tried to use the _count with other methods but it do not works.
 Any help for me please ? Thank you so much for your answers.

**Comment by austincrim** [comment]
Hey @lucasbriandbr 👋,

If you are checking if a `User` has **one or more** `Books`, you can use the `some` option with no parameters: 
```ts
let result = await prisma.user.findMany({
  select: {
    _count: true
  },
  where: {
    posts: {
      some: {}
    }
  }
})
```

You can read more [here](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#some). Let me know if I can clarify anything!

**Reply by leonardomgt** [reply]
What if I wanted to filter for users with >10 posts?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
